### PR TITLE
Increase timeout from 5 to 10 (temporary fix)

### PR DESCRIPTION
**Pull request by krshrimali**
## What does this PR do?

cc: @manskx - Can you help confirm if this is the place where we wanted to increase the timeout?

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue or with the team? (not for typos and docs)
- [ ] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **test your PR locally**?
- [ ] Did you **test your PR on cloud**?

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [ ] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: tests/test_app_gallery.py [hunk 1]
```diff
@@ -201,7 +201,7 @@ def validate_app_functionalities(app_page: "Page") -> None:
         ".MuiGrid-container"
     )
     search_results_container.wait_for(timeout=150 * 1000)
-    sleep(5)
+    sleep(10)
```
Comment (manskx) [inline tests/test_app_gallery.py:204]: @krshrimali 

if we want to wait longer until the result appear, it should be the previous line: 
`search_results_container.wait_for(timeout=150 * 1000)`

we wait 150 seconds now, maybe to increase it a little bit only to 180 ?

Comment (krshrimali) [reply tests/test_app_gallery.py:204]: Ah yes, let me do this.

Comment (krshrimali) [reply tests/test_app_gallery.py]: Done!!

File: tests/test_app_gallery.py [hunk 2]
```diff
@@ -200,7 +200,7 @@ def validate_app_functionalities(app_page: "Page") -> None:
     search_results_container = app_page.frame_locator("iframe").locator(
         ".MuiGrid-container"
     )
-    search_results_container.wait_for(timeout=150 * 1000)
+    search_results_container.wait_for(timeout=180 * 1000)
```
Comment (krshrimali) [inline tests/test_app_gallery.py:203]: @manskx - Does it look good now? :)

**Pull request merged**